src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -869,3 +907,36 @@ function getFeatureValueDefaults(feature: Feature) {
 			return defaults;
 		}, {} as Record<string, string | boolean | undefined>);
 }
+
+export async function getFeatureIdType(output: Log, env: NodeJS.ProcessEnv, id: string) {
+
+	// DEPRECATED: This is a legacy feature-set ID
+	if (!id.includes('/') && !id.includes('\\')) {
+		return { type: 'local-cache', manifest: undefined };
+	}
+
+	// Direct tarball reference
+	if (id.startsWith('http://') || id.startsWith('https://')) {
```

Comment:
We should really disallow `http`. Not sure when we added it and if it is safe to remove without adding backwards compatibility, but `http` is so prone to MITM attacks that I think we should just completely disallow it.

Reply:
definitely worth removing.  I don't believe it will cause any impact and improves the integrity of downloads for a user.